### PR TITLE
Remove Legacy Blockchain documentation and links

### PR DESCRIPTION
**Pull request by samgutentag**

### CODE REVIEW

File: docs/architecture/oracles/oracles.mdx [hunk 1]
```diff
@@ -21,7 +21,7 @@ and the blockchain. These infrastructure changes allow the Helium Network to eff
 new Networks as Decentralized Network Protocol (DNP) subDAOs, and more easily scale existing DNP
 subDAOs (e.g., IOT) by removing existing bottlenecks.
 
-With the deprecation of the Helium L1, changes to the [API](/blockchain/api) and
+With the deprecation of the Helium L1, changes to the API and
```
Comment (jthiller) [inline docs/architecture/oracles/oracles.mdx:24]: #3,4 in the list below needs content updates.

File: docs/architecture/solana/migration/blockchain-api.mdx [hunk 2]
```diff
@@ -10,10 +10,9 @@ slug: /solana/migration/blockchain-api
 
 import useBaseUrl from '@docusaurus/useBaseUrl'
 
-The existing [Helium API](/blockchain/api) will serve only information up to the chain halt at the
-time of migration. Knowing that some users will continue to rely on historical data, The Helium
-Foundation has formed an agreement with the API provider, BlockJoy, to continue hosting this API
-following the migration.
+The existing Helium API will serve only information up to the chain halt at the time of migration.
+Knowing that some users will continue to rely on historical data, The Helium Foundation has formed
+an agreement with the API provider, BlockJoy, to continue hosting this API following the migration.
```
Comment (jthiller) [inline docs/architecture/solana/migration/blockchain-api.mdx]: Needs content updates.

File: netlify.toml [hunk 3]
```diff
@@ -15,12 +15,12 @@
...
 [[redirects]]
   from = "/api"
-  to = "/api/blockchain"
+  to = "/"
   status = 301
   force = true
 [[redirects]]
   from = "/api/blockchain/*"
-  to = "/blockchain/api/:splat"
+  to = "/"
```
Comment (jthiller) [inline netlify.toml:23]: `/blockchain/api/` also needs a redirect.

**Review by jthiller — CHANGES_REQUESTED** [review]
Some content changes to consider.

File: netlify.toml (hunk 3 above)
Comment (samgutentag) [reply netlify.toml:23]: updated to redirect to `/oracles/oracle-data`

File: docs/architecture/oracles/oracles.mdx (hunk 1 above)
Comment (samgutentag) [reply docs/architecture/oracles/oracles.mdx:24]: content added

File: docs/architecture/solana/migration/blockchain-api.mdx (hunk 2 above)
Comment (samgutentag) [reply docs/architecture/solana/migration/blockchain-api.mdx]: minimal update, these migration paged are not intended to last forever as is

**Comment by jthiller** [comment]
Api codebase is still available on GitHub, it just isn't being run to serve L1 data. 
Someone could spin up an instance with a snapshot

**Comment by samgutentag** [comment]
> Api codebase is still available on GitHub, it just isn't being run to serve L1 data.
> Someone could spin up an instance with a snapshot

added link to repo 👍
 
image

**Review by jthiller — APPROVED** [review]

**Pull request merged**